src/PerpetualPositionRouter.sol

```diff
@@ -39,8 +42,9 @@ contract PerpetualPositionRouter {
     TOKEN = asset;
   }
 
-  // TODO: Should we deposit eth into the Perpetual vault?
-  receive() external payable {}
+  receive() external payable {
+    revert UnsupportedFunction();
+  }
```

Comment:
- At first I thought we can just delete this
- Deleting this with a payable fallback results in a solc warning
- So then I changed the fallback to receive, and it errors because you can't   access `msg.data` in receive
- This means the only way is to explicitly revert in the receive function like you have here, and this will only be called if you send ETH with no data.

Is that all correct? If so let's document this as a comment above the receive function, cause it's kind of weird solidity behavior

Reply:
Yes, this is correct I will add a natspec comment